packages/leveldb/src/leveldb/internal.ts

```diff
@@ -231,7 +225,8 @@ export class LevelDBInternal<RxDocType> implements LevelDBStorageInternals<RxDoc
 
                     this.documents.set(id, item)
                     if (!(primaryKeyVal as string).includes("collection") && !(primaryKeyVal as string).includes("storage-token")) {
-                        console.log("Adding ", primaryKeyVal, item)
+                        // console.log("Adding ", primaryKeyVal, item)
+                        // TODO
```

Comment:
I assume this was here for a reason - it's the only thing in that block
so left a `// TODO` in the code. I use this sort of thing to make it easy to find where more work was needed